module_cheatsheets/Bulk_RNA-seq_Cheatsheet.md

```diff
@@ -0,0 +1,63 @@
+|-------------------------------|--------------------------------------------------------------|--------------------------------|--------------------------------------------------------------------------|
+| stringr               | [`word()`](https://www.rdocumentation.org/packages/stringr/versions/1.4.0/topics/word)                    | Word                                            | Extracts words from a character vector                                   |                                                                                        
+| readr                 | [`write_rds()`](https://www.rdocumentation.org/packages/readr/versions/1.3.1/topics/read_rds)             | Write RDS                                       | Writes data to a .rds output file                                        |                                                                                         
+| tximport              | [`tximport()`](https://www.rdocumentation.org/packages/tximport/versions/1.0.3/topics/tximport)           | Tximport                                        | Imports transcript-level estimates and summarizes abundances, counts, and transcript lengths to the gene-level by default   |
+| dplyr                 | [`pull()`](https://www.rdocumentation.org/packages/dplyr/versions/0.7.8/topics/pull)                      | Pull                                            | Extracts a variable as a vector                                          |                                                                                       
+| pheatmap              | [`pheatmap()`](https://www.rdocumentation.org/packages/pheatmap/versions/1.0.12/topics/pheatmap)          | Pheatmap                                        | Plots clustered heatmaps                                                 |                                                   
+
+| Tool               | Piece of Code                                                 | What it's called      | What it does                                                             |
+|-------------------------------|--------------------------------------------------------------|--------------------------------|--------------------------------------------------------------------------|
+| Salmon                | [`salmon index`](https://salmon.readthedocs.io/en/latest/salmon.html)             | Salmon index                                    | Builds a transcriptome index which is required for Salmon quantification (from the command line) |                                                                                        
+| Salmon                | [`salmon quant`](https://salmon.readthedocs.io/en/latest/salmon.html)             | Salmon quantification                           | Runs Salmon’s quantification of transcript expression (from the command line)                    |           
+
+
+### Useful command line commands:
+<br>
```

Comment:
Can you either use examples within the training structure + Docker image or note that this is a departure from that structure you are using for illustrative purposes?

Reply:
@jaclyn-taroni Including headers is a great idea, I'll do that along with the links to their documentation...as well as ensure that all of the links correspond to the package versions used in the training module. 

> Can you either use examples within the training structure + Docker image or note that this is a departure from that structure you are using for illustrative purposes?

Do you have a preference of which route I go here? The examples I used here were meant to be the simplest possible, however, of course there can also be some usefulness in relating them back to training structure.